internal/validator/validator.go

```diff
@@ -51,6 +54,8 @@ type DefaultValidator struct {
 	Validator *validator.Validate
 }
 
+var _ echo.Validator = &DefaultValidator{}
```

Comment:
It's recommended to use this approach if it's only for interface checks.
`var _ echo.Validator = (*DefaultValidator)(nil)`

Reply:
Oops, thanks for letting me know the mistake. I'll fix it.

Reply:
Fixed